src/stellar/StellarTools.ts

```diff
@@ -28,16 +32,18 @@ export class StellarTools implements EdgeCurrencyTools {
   builtinTokens: EdgeTokenMap
   currencyInfo: EdgeCurrencyInfo
   io: EdgeIo
+  log: EdgeLog
   networkInfo: StellarNetworkInfo
 
   highestTxHeight: number = 0
   stellarApiServers: StellarServer[]
 
   constructor(env: PluginEnvironment<StellarNetworkInfo>) {
-    const { builtinTokens, currencyInfo, io, networkInfo } = env
+    const { builtinTokens, currencyInfo, io, log, networkInfo } = env
     this.builtinTokens = builtinTokens
     this.currencyInfo = currencyInfo
     this.io = io
+    this.log = log
```

Comment:
Looks like a testing remnant. Not unwelcome but it's unused and doesn't fit with the commit